src/lib/predictive_model.py

```diff
@@ -1,34 +1,80 @@
 import sklearn
-import lightgbm
-# from sklearn.externals import joblib
 import joblib
 import pandas as pd
 import collections
 import math
 import warnings
 import time
 import os
-import json
-from datetime import datetime
+import torch
+import numpy as np
+import sys
+import lightgbm
+
+sys.path.insert(0, './lib')
+from regression_metrics import mean_absolute_percentage_error
+
 
 class PredictiveModel:
     """
-    Predictive model class is a wrapper for scikit learn regression models
+    Predictive model class is a wrapper for scikit learn regression models and PyTorch.
     ref: http://scikit-learn.org/stable/supervised_learning.html#supervised-learning
+    ref: https://pytorch.org/
     """
 
-    def __init__(self, algorithm, sensor, prediction_horizon, evaluation_periode, error_metrics, split_point,
-                 retrain_period = None, samples_for_retrain = None, retrain_file_location = None):
+    def rmse(self, true, pred):
```

Comment:
There is a file called `regression_metrics.py`. We should include this in there?